packages/DApp/src/constants/contracts.ts

```diff
@@ -1,5 +1,6 @@
 import { ChainId } from '@usedapp/core'
 
+// todo?: move to config.ts
```

Comment:
non-blocking: why it isn't part the config?

Reply:
but I'd expect them to be tied to networks (and their chain definition) which we only support. and if an env like dev/localhost supports multi networks, then it could have multi contract records, each per one. plus I tend to look on the config as sso now